.gitmodules

```diff
@@ -1,3 +1,6 @@
+[submodule "drivers-tools"]
+	path = tests/drivers-tools
```

Comment:
I realize we use this for both Evergreen _and_ GitHub CI, but would it be preferable to keep the path consistent with the repository name?